### DISCUSSION TITLE
Ory Cloud Hydra timeline

### DISCUSSION THREAD

**Discussion by jordanholliday** [ory/hydra#3131] [Q&A] [answered]
Is there a rough timeline for adding Hydra to Ory Cloud? The [Ory Cloud](https://www.ory.sh/cloud) landing page says Hydra is still "coming to Ory Cloud" and from clicking around the Cloud UI, I don't see any Hydra features there.

(apologies if this is the wrong forum for questions like this)

**Comment by aeneasr** [comment, marked as answer]
Around Q3 this year!

**Reply by jordanholliday** [reply]
👍 thanks!
